extensions/ch_vd_sire/views/captage.sql

```diff
@@ -0,0 +1,29 @@
+CREATE VIEW qwat_ch_vd_sire.captage AS
+	SELECT
+		id AS ID_Num
+		, qwat_ext_ch_vd_sire_remarque|| remark AS remarque
+		, NULL AS Date_de_saisie -- TODO
+		, NULL AS Dern_M_a_J -- TODO
+		, precision_code_sire AS Precision_Geo
+		, fk_distributor AS id_distributeur
+		, qwat_ext_ch_vd_sire_etat_exploitation AS Etat_Exploitation
+		, year AS Annee_Construction
+		, NULL::text AS Nom_Descriptif
+		, folder_identification AS Numero_dossier
+		, fk_pressurezone AS ID_Zone_pression
+		, watertype_code_sire AS Type_eau
+		, source_type_code_sire AS Genre
+		, flow_lowest AS Q_Etiage
+		, flow_average AS Q_moyen
+		, source_quality_code_sire AS Qualite_Captage
+		, flow_concession AS Q_concession
+		, contract_end AS Date_fin_Concession
+		, gathering_chamber AS Chambre_de_Rassemblement
+		, NULL AS ID_Qualite -- TODO
+		, altitude AS Altitude
+		, qwat_ext_ch_vd_sire_adesafecter AS A_Desafecter_PDDE
+		, ST_Force2D(geometry) AS geometry
+	FROM qwat_od.vw_export_installation
+	WHERE
+		installation_type = 'source'
```

Comment:
Ne doit-on pas sélectionner uniquement les installations fonctionnelles ou 'en projet' ?

`WHERE installation_type = 'source' AND (status_functional IS TRUE OR fk_status = 1306)`

Si oui, à appliquer aux autres classes

Reply:
`status_active` et `status_functional` quelle différence ici? 

Je filtrerais uniquement par statut : 1301 et 1306 (en service ou projet)

Reply:
oui à discuter, je ne suis pas tout à fait au clair avec les significations de `status_functional` par rapport à ` fk_status = 1301` ou encore `status_active` ... @3nids pourrais-tu nous éclairer ?

Dans mon cas, je peux avoir des objets telles que des conduites qui ont été mises hors service temporairement (ex. travaux), qui certes ne contribuent actuellement pas au réseau mais restent fonctionnels, et devraient donc être transmises au SCAV

Reply:
@kandre va jeter un coup d'oeil à la table qwat_vl.status

https://github.com/qwat/qwat-data-model/blob/master/value_lists/vl_status.sql#L23-L29

Là dedans sont décrites toutes les qualifications d'état. @3nids a fait les choses bien!

Reply:
@3nids @ponceta Ok merci pour ces infos, mais du coup, je vois que la PR ne filtre toujours pas selon le statut

Reply:
pour moi ce n'est pas très clair.
Je pense que la vue doit tout offrir, et c'est peut-être au moment de l'export qu'il faut filtrer par statut et par région géographique.

Reply:
Assez d'accord avec @3nids ici

Reply:
Je vois, car finalement SIRE peut être vu comme un format d'échange, pas uniquement dans le cadre d'un plan directeur PDDE, c'est l'idée ?

Reply:
personnellement, je comprends dans la directive que tous les status existent, ils doivent donc se refléter dans la vue. Après, si le canton ne veut recevoir que de l'actif. Donc, effectivement, ça rejoint ce que tu dis en terme de format d'échange.